feature/experimental/gribi/otg_tests/route_addition_during_failover_test/route_addition_during_failover_test.go

```diff
@@ -0,0 +1,736 @@
+		name := fmt.Sprintf(`dst%d`, i)
+		Index := uint32(i) + 1
+		ateIPv4 := fmt.Sprintf(`198.51.100.%d`, ((4 * i) + 1))
+		dutIPv4 := fmt.Sprintf(`198.51.100.%d`, ((4 * i) + 2))
+		configureSubinterfaceDUT(t, d, dutPort, Index, vlanID, dutIPv4)
+		MAC, _ := incrementMAC(atePort1.MAC, i+1)
+		configureATE(t, top, atePort, vlanID, false, name, MAC, dutIPv4, ateIPv4)
+		nextHops = append(nextHops, ateIPv4)
+	}
+	configureInterfaceDUT(t, dutPort, d, "dst")
+	pushConfig(t, dut, dutPort, d)
+	if *deviations.ExplicitInterfaceInDefaultVRF {
+		intf := d.GetOrCreateInterface(dutPort.Name())
+		for i := 1; i <= nextHopCount+1; i++ {
+			fptest.AssignToNetworkInstance(t, dut, intf.GetName(), *deviations.DefaultNetworkInstance, uint32(i))
+		}
+	}
+	return nextHops
+}
+
+// configureSubinterfaceDUT configures a single DUT layer 3 sub-interface.
+func configureSubinterfaceDUT(t *testing.T, d *oc.Root, dutPort *ondatra.Port, index uint32, vlanID uint16, dutIPv4 string) {
+	t.Helper()
+
+	i := d.GetOrCreateInterface(dutPort.Name())
+	s := i.GetOrCreateSubinterface(index)
+	if vlanID != 0 {
+		if *deviations.DeprecatedVlanID {
+			s.GetOrCreateVlan().VlanId = oc.UnionUint16(vlanID)
+		} else {
+			s.GetOrCreateVlan().GetOrCreateMatch().GetOrCreateSingleTagged().VlanId = ygot.Uint16(vlanID)
+		}
+	}
+
+	sipv4 := s.GetOrCreateIpv4()
+
+	if *deviations.InterfaceEnabled && !*deviations.IPv4MissingEnabled {
+		sipv4.Enabled = ygot.Bool(true)
+	}
+	a := sipv4.GetOrCreateAddress(dutIPv4)
+	a.PrefixLength = ygot.Uint8(uint8(ipv4PrefixLen))
+}
+
+// configureATE configures a single ATE layer 3 interface.
+func configureATE(t *testing.T, top gosnappi.Config, atePort *ondatra.Port, vlanID uint16, addPort bool, Name, MAC, dutIPv4, ateIPv4 string) {
+	t.Helper()
+	if addPort {
+		top.Ports().Add().SetName(atePort.ID())
+	}
+	dev := top.Devices().Add().SetName(Name + ".Dev")
+	eth := dev.Ethernets().Add().SetName(Name + ".Eth").SetMac(MAC)
+	eth.Connection().SetChoice(gosnappi.EthernetConnectionChoice.PORT_NAME).SetPortName(atePort.ID())
+	if vlanID != 0 {
+		eth.Vlans().Add().SetName(Name).SetId(int32(vlanID))
+	}
+	eth.Ipv4Addresses().Add().SetName(Name + ".IPv4").SetAddress(ateIPv4).SetGateway(dutIPv4).SetPrefix(int32(atePort1.IPv4Len))
+
+}
+
+// awaitTimeout calls a fluent client Await, adding a timeout to the context.
+func awaitTimeout(ctx context.Context, c *fluent.GRIBIClient, t testing.TB, timeout time.Duration) error {
+	t.Helper()
+
+	subctx, cancel := context.WithTimeout(ctx, timeout)
+	defer cancel()
+	return c.Await(subctx, t)
+}
+
+// testArgs holds the objects needed by a test case.
+type testArgs struct {
+	ctx        context.Context
+	client     *fluent.GRIBIClient
+	dut        *ondatra.DUTDevice
+	ate        *ondatra.ATEDevice
+	top        gosnappi.Config
+	electionID gribi.Uint128
+}
+
+// createTrafficFlow generates traffic flow from source network to destination network via
+// srcEndPoint to dstEndPoint and checks for packet loss.
```

Comment:
Remove the check for packet loss comment?

Reply:
Removed